app/src/main/kotlin/com/bitwarden/authenticator/data/authenticator/repository/AuthenticatorRepository.kt

```diff
@@ -84,4 +84,10 @@ interface AuthenticatorRepository {
         format: ImportFileFormat,
         fileData: IntentManager.FileData,
     ): ImportDataResult
+
+    /**
+     * Flow that emits `Unit` each time an account is synced from the main Bitwarden app for
+     * the firs time.
```

Comment:
```suggestion
     * the first time.
```

Reply:
Doh! How am I supposed to type two "t"s in a row??

d7256683db1f1d2da6c887d3cd2962ca8feab323